pkgs/tools/networking/opensnitch/daemon.nix

```diff
@@ -39,7 +41,7 @@ buildGoModule rec {
       --replace "/usr/local/bin/opensnitchd" "$out/bin/opensnitchd" \
       --replace "/etc/opensnitchd/rules" "/var/lib/opensnitch/rules" \
       --replace "/bin/mkdir" "${coreutils}/bin/mkdir"
-    sed -i '/\[Service\]/a Environment=PATH=${iptables}/bin' $out/lib/systemd/system/opensnitchd.service
+    sed -i '/\[Service\]/a Environment="PATH=${iptables}/bin:${coreutils}/bin:${iptables}/sbin:${coreutils}/sbin:${findutils}/bin:${systemd}/sbin"' $out/lib/systemd/system/opensnitchd.service
```

Comment:
Please use makeBinPath.

Reply:
ups sorry, this file/change was added by mistake and has nothing to do with the pull request. sorry for that!